### PR TITLE
refactor course blocks

### PR DESCRIPTION
**Pull request by tanyaparker**
- miniblock is now courseblock
- there are three courseblocks: tall, wide-small, and wide-large
- _everything_ about courseblocks is shared in course-blocks.scss
- absolutely no more inline styling in course_wide and course_tall haml views

Examples of tall blocks:

image

image

Examples of wide-small blocks:

image

image

Examples of wide-large blocks:

image

### CODE REVIEW

File: dashboard/app/views/api/k5_courses.html.haml [hunk 1]
```diff
@@ -7,5 +7,5 @@
   = render partial: 'shared/course_wide_block', locals: { id: 'course4' }
 %br/
 .row{style: 'margin: 0px;'}
-  = render partial: 'shared/course_wide_block', locals: { id: '20-hour' }
+  = render partial: 'shared/course_wide_block', locals: { id: "20-hour-large" }
```
Comment (breville) [inline dashboard/app/views/api/k5_courses.html.haml]: heads up: in my change i am changing these ids to use the official ones from script.rb.  might be better to have presentation as a separate parameter to id.

File: dashboard/app/assets/stylesheets/application.css.scss [hunk 2]
```diff
@@ -1011,47 +1011,6 @@ a.download-video {
...
-
-  .miniblock h4 {
-    margin: 0 0 5px 0;
-    font-family: $gotham-bold;
-    float: left;
-    color: $charcoal;
-    font-size: 14px;
-    line-height: 1.2em;
-  }
-
```
Comment (breville) [inline dashboard/app/assets/stylesheets/application.css.scss]: is there more miniblock styling we can take out of this file now?

**Comment by breville** [comment]
good stuff!

File: dashboard/app/assets/stylesheets/application.css.scss (hunk 2 above)
Comment (tanyaparker) [inline dashboard/app/assets/stylesheets/application.css.scss]: I removed everything miniblock.

**Pull request merged**